README.md

```diff
@@ -32,6 +32,19 @@ docker run -it -v $srcDir:/src  <DOCKER_NAME>:<DOCKER_VERSION>
 We use the [codacy-plugins-test](https://github.com/codacy/codacy-plugins-test) to test our external tools integration.
 You can follow the instructions there to make sure your tool is working as expected.
 
+## Update Version with Docs
+
+Change the version in `.brakeman-version` and then run on this project root:
+
+```sh
+bundle clean
```

Comment:
Why do you need to set a custom install path inside the sources? Doesn't it work like https://github.com/codacy/codacy-rubocop#update-version-with-docs ?

Reply:
The script scrapes the source files for additional information

Reply:
Hmmm, then I would prefer to put it in `/tmp`. Putting stuff inside resources is not nice. Also if we do not need to ruby scripts to run we can put then in the root of the project.

Reply:
"if we do not need to ruby scripts to run" i don't get it. what do you mean? they are not added to the docker image.